src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXOverrideMismatch/PXOverrideMismatchAnalyzer.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Immutable;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Utilities.Roslyn.Semantic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Linq;
+using Acuminator.Utilities.DiagnosticSuppression;
+using Acuminator.Utilities.Roslyn.Semantic.PXGraph;
+using Acuminator.Utilities.Common;
+
+namespace Acuminator.Analyzers.StaticAnalysis.PXOverrideMismatch
+{
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class PXOverrideMismatchAnalyzer : PXGraphAggregatedAnalyzerBase
+	{
+		public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics =>
+			ImmutableArray.Create(Descriptors.PX1096_PXOverrideMustMatchSignature);
+
+		public override bool ShouldAnalyze(PXContext pxContext, PXGraphSemanticModel graph) =>
+			base.ShouldAnalyze(pxContext, graph) && graph.Type == GraphType.PXGraphExtension;
+
+		public override void Analyze(SymbolAnalysisContext context, PXContext pxContext, PXGraphSemanticModel pxGraphExtension)
+		{
+			context.CancellationToken.ThrowIfCancellationRequested();
+
+			pxGraphExtension.Symbol
+				.GetMembers()
+				.OfType<IMethodSymbol>()
+				.Where(m => !m.IsStatic && m.HasAttribute(pxContext.AttributeTypes.PXOverrideAttribute, false))
+				.ForEach(m => AnalyzeMethod(context, pxContext, m));
+		}
+
+		private void AnalyzeMethod(SymbolAnalysisContext context, PXContext pxContext, IMethodSymbol methodSymbol)
+		{
+			// Here we know that the method is not static and has the correct attribute.
+
+			var extensionType = methodSymbol.ContainingType.GetPXGraphExtension(pxContext.PXGraphExtension.Type);
+			var graphType = extensionType.GetFirstTypeArgument();
```

Comment:
The helper is incorrect like I wrote before. Moreover it is not needed, Pass the graph semantic model to this method, it has a separate graph symbol

Reply:
We use GetGraphExtensionWithBaseExtensions now, these helpers are removed. Thank you!